src/command_modules/azure-cli-managementpartner/azure/cli/command_modules/managementpartner/_help.py

```diff
@@ -0,0 +1,51 @@
+﻿# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+helps['managementpartner'] = """
+    type: group
+    short-summary: Manage Azure ManagementPartner.
+"""
+
+helps['managementpartner create'] = """
+    type: command
+    short-summary: Create new management partner
+    long-summary: Add a management partner id to the current user
+    parameters:
+        - name: --partner-id
+          type: string
+          short-summary: Id of management partner
```

Comment:
Instead of this, you could just have a single param registration `c.argument('partner_id', help='Microsoft partner network ID..."` in your params.py file. Otherwise, if you want to change this for all commands you have to update it in multiple places.